core/src/executor/alter/table.rs

```diff
@@ -102,7 +102,7 @@ pub async fn create_table<T: GStore + GStoreMut>(
                 Some(column_defs)
             }
         },
-        None if !column_defs.is_empty() => Some(column_defs.to_vec()),
+        None if column_defs.is_some() => column_defs.map(|s| s.to_vec()),
```

Comment:
is there any way like `.map(slice::to_vec)`?

Reply:
that's interesting.
I just got to know that.. this works!
```rust
 column_defs.map(<[_]>::to_vec)
```